Emotion/Android/ClientLibrary/lib/src/main/java/com/microsoft/projectoxford/emotion/contract/Scores.java

```diff
@@ -41,4 +49,54 @@
     public double neutral;
     public double sadness;
     public double surprise;
+    
+    public List<Map.Entry<String, Double>> ToRankedList(Order order)
+    {
+		
+	// create a Map to store each entry
```

Comment:
Is there an issue with indentation?  Our convention is to indent with spaces instead of tabs.